addons/sale/migrations/11.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def update_field_module(cr):
```

Comment:
I think this should be a function in openupgradelib.

Reply:
@pedrobaeza how should I proceed, create a PR in openupgradelib with this function?

Reply:
Yes!

Reply:
@pedrobaeza : I created a new method to handle this case in openupgradelib (https://github.com/OCA/openupgradelib/pull/92). After this is merged, I will update this.